developer/tutorial/tutorial-debugging-server-code.md

```diff
@@ -0,0 +1,155 @@
+# Tutorial: Debugging server-side code
+
+As of Meteor 1.6, you can now debug server code using the `--inspect` flag in Chrome DevTools and within integrated developer environments like [Visual Studio Code (VS Code)](https://code.visualstudio.com/), a free code editor, and [WebStorm by JetBrains](https://www.jetbrains.com/webstorm/).
+
+Here are the steps to get started using Reaction in `inspect` mode in any editor:
+
+## Launch the application in `inspect` mode
+
+1. Before we get started, make sure you have at least Reaction 1.6.
+
+You can do that by running:
+
+```sh
+reaction -v
+```
+
+You should see an output of Reaction's version:
+
+```sh
+Node: 8.2.1
+NPM: 5.6.0
+Meteor Node: 8.9.3
+Meteor NPM: 5.5.1
+Reaction CLI: 0.24.2
+Reaction: 1.6.4
+Reaction branch: master
+Docker: 17.09.0-ce
+```
+
+If your Reaction version is older than 1.6, you will have to upgrade to at least 1.6 first.
+
+2. Instead of using `reaction` to run the application, use the Meteor [`run`](https://docs.meteor.com/commandline.html#meteorrun) method with the following flags:
```

Comment:
People can still run `reaction` and pass these additional parameters I believe. Otherwise the plugin loader will not run which might cause confusion.

Reply:
@zenweasel I can't run `reaction run --settings settings/dev.settings.json --raw-logs --inspect` I believe it is not supported by the CLI yet, right @jshimko ?

Reply:
I'm able to run it w/o problems. What are you running into?

````
 »» reaction run --settings settings/settings.json --raw-logs --inspect

Using settings file at settings/settings.json

Setting up plugin imports...

Installing dependencies for connectors-shopify...

up to date in 0.057s
Setting up style imports...

[[[[[ ~/Projects/js/reaction ]]]]]

=> Started proxy.
=> Started MongoDB.
04:45:07.841Z  INFO Reaction: Load default data from /private/data/
04:45:08.052Z  INFO Reaction: JobServer started
04:45:08.072Z  INFO Reaction: Loaded "/private/settings/reaction.json" for registry fixture import
Debugger listening on ws://127.0.0.1:9229/b37b7c73-f34e-4df2-91d7-a6f5ce751c33
For help see https://nodejs.org/en/docs/inspector
Note: you are using a pure-JavaScript implementation of bcrypt.
While this implementation will work correctly, it is known to be
approximately three times slower than the native implementation.
In order to use the native implementation instead, run

  meteor npm install --save bcrypt

in the root directory of your application.
04:45:09.234Z  INFO Reaction: Reaction Version: 1.6.4
04:45:09.605Z  INFO Reaction: Reaction initialization finished.
=> Started your app.

=> App running at: http://localhost:3000/
```

Reply:
Ooh - I got mine to work too. Will update. Thank you.